### DISCUSSION TITLE
build A380

### DISCUSSION THREAD

**Discussion by c0ball** [flybywiresim/aircraft#8795] [Help] [answered]
I know that the development for the A380 is not done yet, but is it possible to build it from source to play around with it?

**Comment by alepouna** [comment, marked as answer]
You can build it, but without a model (it is not publicly available)

**Reply by c0ball** [reply]
Thanks for the reply, is there any chance to receive the model or a build with the model?
 Do you have an estimate of when the model will be released? 🥲

**Reply by alepouna** [reply]
Sorry but we are not ready to distribute it yet. You will have to use another skeleton (e.g. the A32NX) if you want to mess with the instruments & systems. 

As for 'when': 
> Due to the community-driven nature of our projects, we do not enforce delivery timelines on our volunteer developers. We value high-quality, polished releases over frequent and quickly-released ones, which means some features might take time to land in a publicly-available build. Rest assured that our developers and contributors are working hard on making every FlyByWire project as bug-free and feature-complete as possible, but that we are not able to provide release dates or estimates for a certain feature/fix.

**Reply by c0ball** [reply]
I totally understand that, thanks for your help.
